tsl/src/init.c

```diff
@@ -235,6 +236,7 @@ CrossModuleFunctions tsl_cm_functions = {
 	.cache_syscache_invalidate = cache_syscache_invalidate,
 	.update_compressed_chunk_relstats = update_compressed_chunk_relstats,
 	.health_check = ts_dist_health_check,
+	.mn_set_foreign_join_paths = ts_get_foreign_join_paths,
```

Comment:
Can you use the same name for the field? Easier to grep.

Reply:
The name of the function is changed.